File: Dia17/OrdenarMayorMenor.py
#Ordenar una lista de números de mayor a menor en javaScript

#Funcion para ordenar una lista de números de mayor a menor
def sortHL(numbers):
    numbers.sort(reverse=True)
    return numbers

#Pedido de datos
cantidad = int(input('¿Cuántos números quiere ordenar de mayor a menor?: '))
numeros = []
for i in range(0, cantidad):
    numeros.append(float(input('Introduzca el número ({}/{}): '.format(i+1, cantidad))))

#Impresion de resultado
print('Números ordenados: {}'.format(sortHL(numeros)))

File: Dia17/OrdenarMayorMenor.js
//Ordenar una lista de números de mayor a menor en javaScript

//Funcion para ordenar una lista de números de mayor a menor
function sortHL(numbers) {
    numbers.sort((a, b) => b - a)
    return numbers
}

//Pedido de datos
const cantidad = parseInt(prompt('¿Cuántos números quiere ordenar de mayor a menor?'))
const numeros = []
for (let i = 0; i < cantidad; i++) {
    numeros.push(parseFloat(prompt(`Introduzca el número (${i+1}/${cantidad})`)));
}

//Impresion de resultado
alert(`Números ordenados: ${numeros}`)